bpfman/src/lib.rs

```diff
@@ -93,7 +93,116 @@ pub(crate) fn get_db_config() -> SledConfig {
     SledConfig::default().temporary(true)
 }
 
-/// Loads an ebpf program.
+/// Adds an eBPF program to the system.
+///
+/// This function takes a `Program` and performs the necessary loading
+/// and attaching operations to add it to the system. It supports
+/// various types of eBPF programs such as XDP, TC, Tracepoint,
```

Comment:
Globally, go ahead and add TCX everywhere you mention TC, because it's coming and code has already started, so they will have missed your changes when grepping through the code for TC.

Reply:
For the commentary I've added, I believe this is the only reference to TC. Updated in: f66f909491cb79fd1d4890b1a8a6d843fda8b877.